### ISSUE TITLE
getConnectWidgetUrl should handle configuration options

### ISSUE THREAD

**Issue by ckknight** [mxenabled/mx-atrium-node#12]
As per https://atrium.mx.com/documentation#mx-connect-configuration-options

There is currently no way to pass the body in when calling `getConnectWidgetUrl`.

**Comment by ronaldnjus18** [comment]
Correct. Instead, if you really wanted pass in options after you loaded the library, you should use:

Atrium._fetchUtility('/users/:'+request.params.userGuid+'/connect_widget_url',
     'POST',
     {'is_mobile_webview':true})
.then(...)
.catch(...)

Note, the above is almost how the _fetchUtility is called by the getConnectWidgetUrl is called.

**Comment by ckknight** [comment]
It seems inappropriate to use private (read: prefixed-with-`_`) methods when it could be better-handled by accepting another argument.

Minimally, the appropriate and expected way to accomplish this should be documented somewhere.
